components/family-studies/ui/src/main/resources/PhenoTips/FamilySelector.xml

```diff
@@ -144,15 +144,15 @@
             this.createFamily.observe('click', this._onOptionSelected.bind(this) );
             this.assignFamily.observe('click', this._onOptionSelected.bind(this) );
 
-            createLabel.insert(this.createFamily).insert("Create a new family");
+            createLabel.insert(this.createFamily).insert("$services.localization.render('phenotips.UIXField.family_membership.create')");
```

Comment:
Please always use `"$!escapetool.javascript($services.localization.render('...'))"` to prevent any special characters that may occur in translated strings from breaking the whole script.